zerver/lib/stream_topic.py

```diff
@@ -32,5 +36,22 @@ def user_ids_muting_topic(self) -> Set[int]:
             for row in query
         }
 
+    def get_user_ids_muting_stream(self) -> Set[int]:
+        subscription_rows = Subscription.objects.filter(
+            recipient__type=Recipient.STREAM,
+            recipient__type_id=self.stream_id,
+        ).distinct('user_profile_id').values('user_profile_id', 'is_muted')
```

Comment:
Should we just include `is_muted=True` in the filter, and do `.values_list("user_profile_id", flat=True)` here?  Then we wouldn't need the next lines.